CHANGELOG.md

```diff
@@ -20,6 +87,8 @@ The first Python 3 ONLY capa version.
 - py3: drop Python 2 support #480 @Ana06
 - deps: bump ruamel yaml parser to 0.17.4 #519 @williballenthin
 - explorer: explain how to install IDA 7.6 patch to enable the plugin #528 @williballenthin
+- explorer: document IDA 7.6sp1 as alternative to the patch #536 @Ana06
+- rules: update ATT&CK and MBC mappings https://github.com/fireeye/capa-rules/pull/317 @williballenthin
```

Comment:
I'm not sure the change log should include rules updates beyond the new rules listed above.

Reply:
I think significant changes (either because they imply a lot of work and want to recognize the author or because they introduce import changes) should be included (there are already similar entries in the changelog). This is the only part that would need to be done manually for capa rules. But I don't expect those changes to be that many. I am not sure if this change is that import though.

Other thoughs on this? @williballenthin @mike-hunhoff @re-fox

Reply:
Continue discussion here: https://github.com/fireeye/capa/issues/550, I am going to merge this 😉